nvi

```diff
@@ -1,12 +1,21 @@
 #!/bin/bash
 
+set -o errexit
+set -o nounset
+set -o pipefail
+
 NVI_VERSION=1.0.0
 B=$(tput bold) # Bold
 N=$(tput sgr0) # Normal
 U=$(tput smul) # Underline
 
-function print_usage() {
-  cat << EOF
+print_usage() {
+  less \
+    --raw-control-chars \
```

Comment:
I think the readability could benefit in having the flags sorted alphabetically to allow for an easier lookup of whether or not a flag is present.